src/main/webapp/app/exercises/programming/manage/instructions-editor/programming-exercise-editable-instruction.component.ts

```diff
@@ -279,7 +279,7 @@ export class ProgrammingExerciseEditableInstructionComponent implements AfterVie
     loadTestCasesFromTemplateParticipationResult = (templateParticipationId: number): Observable<Array<string | undefined>> => {
         // Fallback for exercises that don't have test cases yet.
         return this.programmingExerciseParticipationService.getLatestResultWithFeedback(templateParticipationId).pipe(
-            rxMap((result) => (!result || !result.feedbacks ? throwError('no result available') : result)),
+            rxMap((result) => (!result || !result.feedbacks ? throwError(() => 'no result available') : result)),
```

Comment:
```suggestion
            rxMap((result) => (!result || !result.feedbacks ? throwError(() => new Error('no result available')) : result)),
```
Are you sure this works that way? I thought it should be my suggested way according to the documentation.

Reply:
If you change it, please change other occurrences as well.